org.polymc.PolyMC.yml

```diff
@@ -28,6 +28,17 @@ finish-args:
   - --env=PATH=/app/bin:/app/utils/gamescope/bin:/usr/bin
 
 modules:
+  # Git, closes https://github.com/flathub/org.polymc.PolyMC/issues/21
+  - name: git
+    buildsystem: simple
+    build-commands:
+      - make prefix=/app install
+    sources:
+      - type: git
+        url: https://github.com/git/git.git
+        commit: ac8035a2affdf30f2c691ad760826d955bba0507
```

Comment:
1. Use tags
2. Use flatpak external data checker for automatic updates

Git frequently has CVE's that require immediate reaction. It must be trivially knowable if the app contains vulnerable version.

Reply:
See https://github.com/flathub/flatpak-external-data-checker for 2